src/pages/Collections/CollectionsPage.svelte

```diff
@@ -1,14 +1,96 @@
 <script lang="ts">
   import { Route } from "svelte-navigator";
   import { Pane, Splitpanes } from "svelte-splitpanes";
-
+  // ---- Store
   import {
     collectionRightPanelWidth,
     collectionLeftPanelWidth,
     collapsibleState,
+    userWorkspaceLevelRole,
   } from "$lib/store";
+  import type { Writable } from "svelte/store";
 
+  // ---- Components
   import RestExplorer from "../RestExplorer/RestExplorer.svelte";
+  import TabBar from "$lib/components/collections/tab-bar/TabBar.svelte";
+  import CloseConfirmationPopup from "$lib/components/popup/CloseConfirmationPopup.svelte";
+  import { notifications } from "$lib/components/toast-notification/ToastNotification";
+
+  // ---- Interface, enum & constants
+  import type { NewTab } from "$lib/utils/interfaces/request.interface";
+  import type { WorkspaceRole } from "$lib/utils/enums/team.enum";
+  import { workspaceLevelPermissions } from "$lib/utils/constants/permissions.constant";
+
+  // ---- View Model
+  import { CollectionPageViewModel } from "./CollectionPage.ViewModel";
+
+  // ---- helpers
+  import { hasWorkpaceLevelPermission } from "$lib/utils/helpers";
+
+  const _collectionPageViewModel = new CollectionPageViewModel();
+  const tabList: Writable<NewTab[]> = _collectionPageViewModel.tabs;
+  let loggedUserRoleInWorkspace: WorkspaceRole;
+  let removeTab: NewTab;
+  let closePopup: boolean = false;
+  let saveAsVisibility: boolean = false;
+  let loader = false;
+
+  /**
+   * This function is to handle close tab functionality in tab bar list
```

Comment:
We can make it more precise. We are adding unnecessary prefix "this function is to ...." in all of our comments.

The comment can be like this
"Handles close tab functionality in tab bar"

Reply:
Done